assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -20,7 +20,7 @@
 {{- define "token" -}}
 {{ include (print $.Template.BasePath "/kube-apiserver-secret.yaml") . | sha256sum }}
 {{- end -}}
-{{- define "authHeader" }}
+{{- define "authHeader" -}}
```

Comment:
Can you please elaborate on the commit? Removing the trailing blank spaces why fixes it?

Reply:
Without this patch, the template content looks like following:
```

...
```

So when you inject it into the YAML, it looks like:
```yaml
foo: "
...
```

Instead of:
```yaml
foo: "..."
```

I'm not sure how to inject such details into commit messages. It's hard to describe, but easy to see if you look at the manifests. I should perhaps do a diff on rendered file :thinking:

Reply:
You can just put this in the commit message, IMHO :-)

Reply:
I added my comment to commit message, not sure if it's very readable, but it's something I guess :smile: